packages/core/src/components/alert/alert.stories.ts

```diff
@@ -0,0 +1,85 @@
+import { StoryFn } from '@storybook/addons';
+import { storiesOf } from '@storybook/web-components';
+import { StoryFnHtmlReturnType } from '@storybook/web-components/dist/ts3.9/client/preview/types';
+import { html } from 'lit-html';
+
+// @ts-ignore
```

Comment:
Dit is nodig om een onterechte melding in IDE te omzeilen:

![image](https://user-images.githubusercontent.com/9696261/114368164-6ef99600-9b7d-11eb-8eb4-9bce3512a47e.png)

De `tsconfig.json` van Storybook's webpack loopt niet synchroon met de `tsconfig.json` van deze workspace